services/121-service/package.json

```diff
@@ -100,6 +99,7 @@
     "supertest": "^7.0.0",
     "ts-jest": "^29.1.2",
     "tsc-watch": "^6.2.0",
+    "typescript": "^5.5.0-beta",
```

Comment:
I'd be a bit squeamish about using a 'beta' version... but if it really unlocks features we need right now, or speeds things up...

Reply:
Normally, me too. I'm a bit reassured by the fact that:
- TS team says it can be used in production
- I'd assume that any weirdness stemming from a TS beta would be caught either in compilation or in starting the service
- This version is [set to release officially in about a month](https://github.com/microsoft/TypeScript/issues/57475) so we wouldn't be in beta for long

> but if it really unlocks features we need right now, or speeds things up...

Indeed, not necessarily that we "need" but it does avoid adding a lot of extra code that would be made redundant with the 5.5 version